client/layout/nps-survey-notice/index.jsx

```diff
@@ -58,6 +59,9 @@ class NpsSurveyNotice extends Component {
 			// (2) the user notices the notice more, since it will cause a change to the
 			//     screen they are already looking at
 			setTimeout( this.props.showNpsSurveyNotice, 3000 );
+
+			analytics.mc.bumpStat( 'calypso_nps_survey', 'displayed' );
+			analytics.tracks.recordEvent( 'calypso_nps_survey_displayed' );
```

Comment:
Let's rename this to 'calypso_nps_notice_displayed', to make it clearer that it is the notice, not the actual survey, that we are talking about.